docs/userGuide/siteConfiguration.md

```diff
@@ -82,6 +88,7 @@ Here is a typical `site.json` file:
 * **`title`**: The page `<title>` for the generated web page. Titles specified here take priority over titles specified in the [front matter](addingPages.html#front-matter) of individual pages.
 * **`layout`**: The [layout]({{ baseUrl }}/userGuide/advancedTopics.html#page-layout) to be used by the page. Default: `default`.
 * **`searchable`**: Specifies that the page(s) should be excluded from searching. Default: `yes`.
+* **`externalScripts`**: An array of external scripts to be refrenced on the page.

```

Comment:
refrenced -> referenced

Reply:
As discussed, also maybe describe where the scripts are inserted on the page, right now it is not very obvious.

Reply:
I added a brief description! Let me know if that's okay.